### ISSUE TITLE
Error Message at List view if Fe User not logged in

### ISSUE THREAD

**Issue by enilee** [oliverklee/ext-seminars#29]
If you not logged in as FE User, you see that Error Message.
Is there a configuration for it, or is it really a mistake?

> Uncaught TYPO3 Exception
> #1331488761: $uid must be > 0. (More information)
> InvalidArgumentException thrown in file
> /typo3conf/ext/oelib/Classes/IdentityMap.php in line 72.

![image](https://user-images.githubusercontent.com/17785409/38072739-00bc4fea-3328-11e8-87e0-ac25573a253a.png)

**Comment by oliverklee** [comment]
Is this the normal list view, or another list, e.g., the "my events" list?

**Comment by enilee** [comment]
normal list view

**Comment by oliverklee** [comment]
On my machine on TYPO3 7.6, the list view displays fine without being logged in as an FE user.

Would you be willing to debug this, e.g. to get a stack trace?

**Comment by enilee** [comment]
the error only occurs in my local installation
.
The installations, htaccess, Flexform etc are versioned and therefore identical.
Do you have any idea why the error occurs?

**Comment by oliverklee** [comment]
No, unfortunately not. You'll need to debug this on your installation and find out why there is a UID of 0 or < 0.

**Comment by oliverklee** [comment]
@enilee Which TYPO3 version are you using?

**Comment by oliverklee** [comment]
https://github.com/oliverklee/ext-oelib/pull/17 will probably fix this.

**Comment by enilee** [comment]
okay, the error occurs not only in my local installation. 
I´ve updated oelib.

Here the complete Error, i hope it helps:

[error.pdf](https://github.com/oliverklee/ext-seminars/files/1898089/error.pdf)

**Comment by oliverklee** [comment]
Hi @enilee, I haven't fixed it in oelib yet. So an oelib update will not help (yet).

Which versions of TYPO3 and seminars are you using?

**Comment by enilee** [comment]
yes ive seen it.

TYPO3: 8.7.10
Seminars: 1.4.1

**Comment by enilee** [comment]
Hello Oli,

I have observed the behavior when the error message occurs.
It is different. Sometimes this occurs locally, but not on the test web server.
Sometimes the other way round too.

Today, the error on the test server but not locally.

I have integrated your bugfix, deleted all caches and the bug is fixed.
I will keep watching if it has any other effects.

Thank you for your quick bug fix.

Best regards
